templates/internet-of-things/contact-us-hannover.html

```diff
@@ -0,0 +1,222 @@
+              <input required id="lastName" name="lastName" maxlength="255" type="text" />
+            </li>
+            <li class="p-list__item">
+              <label for="company">Company:</label>
+              <input required id="company" name="company" maxlength="255" type="text" />
+            </li>            
+            <li class="p-list__item">
+              <label for="email">Email:</label>
+              <input required id="email" name="email" maxlength="255" type="email" pattern="^[^ ]+@[^ ]+\.[a-z]{2,26}$" />
+            </li>
+            <li class="p-list__item">
+              <label for="phone">Mobile/cell phone number:</label>
+              <input required id="phone" name="phone" maxlength="255" type="tel" />
+            </li>
+          </ul>
+        </fieldset>
+        
+        <fieldset class="u-no-margin--bottom">
+          <ul class="p-list">
+            <li class="p-list__item u-hide">
+              <label for="Comments_from_lead__c">What would you like to talk to us about?</label>
+              <textarea id="Comments_from_lead__c" name="Comments_from_lead__c" rows="5" maxlength="2000"></textarea>
+            </li>
+            <li class="p-list__item">
+              <label class="p-checkbox">
+                <input class="p-checkbox__input" value="yes" aria-labelledby="canonicalUpdatesOptIn" name="canonicalUpdatesOptIn" type="checkbox">
+                <span class="p-checkbox__label" id="canonicalUpdatesOptIn">I agree to receive information about Canonical's products and services.</span>
+              </label>
+            </li>
+            <li class="p-list__item">In submitting this form, I confirm that I have read and agree to <a href="/legal/data-privacy/contact">Canonical's Privacy Notice</a> and <a href="/legal/data-privacy">Privacy Policy</a>.</li>
+            {# These are honey pot fields to catch bots #}
+            <li class="u-off-screen">
+              <label class="website" for="website">Website:</label>
+              <input name="website" type="text" class="website" autocomplete="off" value="" id="website" tabindex="-1" />
+            </li>
+            <li class="u-off-screen">
+              <label class="name" for="name">Name:</label>
+              <input name="name" type="text" class="name" autocomplete="off" value="" id="name" tabindex="-1" />
+            </li>
+            {# End of honey pots #}
+            <li class="p-list__item">
+              <button type="submit" class="p-button--positive" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Form', 'eventAction' : 'hannover contact-us', 'eventLabel' : 'hannover', 'eventValue' : undefined });">Submit</button>
+            </li>
+          </ul>
+          <input type="hidden" aria-hidden="true" aria-label="hidden field" name="formid" value="4479" />
+          <input type="hidden" aria-hidden="true" aria-label="hidden field" name="returnURL" value="/internet-of-things/contact-us-hannover" />
```

Comment:
Your just returning to the same page with no comfirmation

Reply:
The Pdm asked for the form to return to it's self after submission, as it is to be used at an event on a tablet, with this as the only page.

Reply:
Not sure I'm following what's happened with these comments - Ant left a reply, that was then edited by Pete to reflect a conversation I had with Pete?

Reply:
Not sure why it was edited but there is zero confirmation to the user and I think as a team we should encourage good user experience.

Reply:
I replied through my phone, I must have clicked the wrong thing, apologies.

Are you suggesting we include a thank you page, and just manually reset the form each time?

Reply:
I'd suggest showing a notification like we do with dynamic forms - when the submission is successful, it's redirected back to the original page with `#success` in the URL. Then, since we know we want the form to be ready for the next user with minimal intervention, hide the message after a certain amount of time, maybe 10-15 seconds.

Reply:
If we know it is for circular use (next person at the conf) we could dismiss after a few seconds.